### DISCUSSION TITLE
class as per media size not working

### DISCUSSION THREAD

**Discussion by Uddhav-Paudel** [themesberg/flowbite#340] [Q&A] [answered]
<NavbarLink class="**md:py-2**" is-active>Home</NavbarLink>

md:py-2 class is not reflecting the padding for medium sized screens.

FYI: Language used is vue.

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hello @Uddhav-Paudel,

Please refer to the new [Flowbite Vue 3](https://flowbite.com/docs/getting-started/vue/) guide and update to `v1.6.0`.

You also need to make sure that the template paths are correctly set in your `tailwind.config.js` file.

Cheers,
Zoltan
